website/content/docs/platform/k8s/helm/enterprise.mdx

```diff
@@ -46,6 +48,38 @@ Once the cluster is [initialized and unsealed](/docs/platform/k8s/helm/run), you
 kubectl exec -ti vault-0 -- vault license get
 ```
 
+### License Update
+
+To update the autoloaded license in Vault, you may do the following:
+
+- Delete and recreate the secret
+
+  Use the name specified in chart option [`server.enterpriseLicense.secretName`](/docs/platform/k8s/helm/configuration#secretname-1). Updating the existing secret will work as well.
+
+```shell
+kubectl delete secret vault-ent-license
+secret=$(cat new-license.hclic)
+kubectl create secret generic vault-ent-license --from-literal="license=${secret}"
+```
+
+- Wait until [`vault license inspect`](/docs/commands/license/inspect) shows the updated license
+
+```shell
+kubectl exec -ti vault-0 -- vault license inspect
+```
+
+- Reload Vault's config by issuing an HUP signal
```

Comment:
There's also an endpoint now to do license reload.  I suspect it's less convenient than using sighup here, since you'd have to use a token.  I just mention it in case it's helpful.

Reply:
Good point, I do think that's worth mentioning here. Updated in d3e7aa8183defd0e013a20774ea3f32a3965550d.